modules/twinkleblock.js

```diff
@@ -433,9 +433,37 @@ Twinkle.block.callback.change_action = function twinkleblockCallbackChangeAction
 			tags: true,
 			placeholder: 'Enter pages to block user from',
 			maximumSelectionLength: 10, // Software limitation [[phab:T202776]]
-			language: {
-				noResults: function() {
-					return 'No pages entered yet';
+			minimumInputLength: 1, // prevent ajax call when empty
```

Comment:
1 is the pretty much the universal default on such input fields. And also since we aren't allowing selection of titles not returned by the API, that would make  it impossible to select pages with short names.

Reply:
Fair enough, and I like the delay option better anyway.